benches/bench.rs

```diff
@@ -38,15 +74,40 @@ mod poly_benches {
                 let rand_samples = || (0..=deg).map(|i| (i, rng.gen::<Fr>())).collect::<Vec<_>>();
                 b.iter_with_setup(rand_samples, Poly::interpolate)
             },
-            &[5, 10, 20, 40],
+            &TEST_DEGREES,
         );
     }
 
     criterion_group!{
         name = poly_benches;
         config = Criterion::default();
-        targets = multiplication, interpolate,
+        targets = multiplication, interpolate, subtraction, addition,
+    }
+}
+
+mod bivar_poly_benches {
+    use super::*;
+
+    // Benchmarks the generation of random bivariate polys
+    fn random(c: &mut Criterion) {
```

Comment:
I'm not sure if this benchmark is necessary, it basically will just tell us how fast `rng.gen()` can fill up a `Vec<Fr>` with `deg` number of elements. All `BivarPoly::random()` does is create a vector, wrap it in a `BivarPoly`, then return.